acct-user/automx2/automx2-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for www-servers/automx2"
+
+ACCT_USER_ID=437
+ACCT_USER_GROUPS=( automx2 )
+ACCT_USER_HOME="/var/lib/automx2"
```

Comment:
Do you really need this? Have you read https://devmanual.gentoo.org/ebuild-writing/users-and-groups/index.html and "Choosing a home directory" part?

Reply:
automx2 requires a database, which in the simplest form uses SQLite. The example configuration I provided stores that DB in `/var/lib/automx2`.